v20.2/joins.md

```diff
@@ -131,7 +131,7 @@ Hash joins are performed on two tables as follows:
 
 ### Lookup joins
 
-The [cost-based optimizer](cost-based-optimizer.html) decides when it would be beneficial to use a lookup join. Lookup joins are used when there is a large imbalance in size between the two tables, as it only reads the smaller table and then looks up matches in the larger table. A lookup join requires that the right-hand (i.e., larger) table is indexed on the equality column.
+The [cost-based optimizer](cost-based-optimizer.html) decides when it would be beneficial to use a lookup join. Lookup joins are used when there is a large imbalance in size between the two tables, as it only reads the smaller table and then looks up matches in the larger table. A lookup join requires that the right-hand (i.e., larger) table is indexed on the equality column, or [partially indexed](partial-indexes.html) on the subset of rows in the equality column.
```

Comment:
[nit] `on the subset of rows in the equality column` is confusing to me. Maybe something like:

> A lookup join requires that the right-hand (i.e., larger) table is indexed on the equality column. A [partial index](partial-indexes.html) can only be used if it contains the subset of rows being looked-up.